person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :age, :name
+  attr_reader :id
```

Comment:
- [ ] For better code organization I would recommend you move these getters and setters to the top of the class.